js/script.js

```diff
@@ -56,21 +56,22 @@ document.getElementById('email').appendChild(email)
 if ( OS == 'Android' || OS == 'iOS' ) {
     var btn = document.createElement("button")
     btn.textContent = "Enroll Device"
-    btn.addEventListener('click', function() {
+    var clickEvent = ((document.ontouchstart!==null)?'click':'touchstart')
+    btn.addEventListener(clickEvent, function() {
         window.location.replace(deeplink)
         var cont = 0
         var timer = setInterval( function()  { 
             cont ++
             if (document.hidden) clearInterval(timer)
```

Comment:
Why not use the event listener?
`document.addEventListener(visibilityChange`...

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Document/hidden